site/community.md

```diff
@@ -31,5 +31,5 @@ and "why does Contour do this thing this way?"
 [3]: {{site.footer_social_links.Twitter.url}}
 [4]: {{site.footer_social_links.Slack.url}}
 [5]: https://hackmd.io/84Xbl4WBTpm7OBhaOAsSiw
-[6]: https://www.youtube.com/playlist?list=PL7bmigfV0EqTBsPrnCkzhu0R4SAWnBjLj
+[6]: https://www.youtube.com/playlist?list=PLk2K7YhXu5KtYU1UGEYNC8ApH5gsWihDJ
```

Comment:
@jonasrosland why does this link not include the latest meetings?

Reply:
Updated, they should show up there now (or whenever YouTube finishes the processing)